test/nullable.jl

```diff
@@ -102,6 +102,8 @@ for (i, T) in enumerate(types)
     showcompact(io2, get(x3))
     @test String(take!(io1)) == @sprintf("Nullable{%s}(%s)", T, String(take!(io2)))
 
+    # these tests are disabled rather than fixed, because Nullable is to be removed from base
```

Comment:
Just leaving a note so that we don't forget it.

Reply:
What do you mean?

Reply:
Just that I left that comment so that it's obvious to everybody that this shouldn't be merged until `Nullable` has been removed so that we don't leave these commented out tests.